src/libmongoc/tests/test-mongoc-structured-log.c

```diff
@@ -714,4 +821,11 @@ test_structured_log_install (TestSuite *suite)
    TestSuite_Add (suite, "/structured_log/duration", test_structured_log_duration);
    TestSuite_Add (suite, "/structured_log/level_names", test_structured_log_level_names);
    TestSuite_Add (suite, "/structured_log/component_names", test_structured_log_component_names);
+   TestSuite_Add (suite, "/structured_log/max_document_length", test_structured_log_max_document_length);
+   TestSuite_AddFullWithTestFn (suite,
+                                "/structured_log/env_defaults",
+                                TestSuite_AddHelper,
+                                NULL,
+                                test_structured_log_env_defaults,
+                                check_test_structured_log_env_defaults);
```

Comment:
```suggestion
   TestSuite_AddFull (suite,
                      "/structured_log/env_defaults",
                      test_structured_log_env_defaults,
                      NULL,
                      NULL,
                      check_test_structured_log_env_defaults);
```
Recommend using `TestSuite_AddFull` with `_dtor` and `_ctx` set to `NULL` due to no test context data being required. This avoids the need to expose `TestSuite_AddHelper`.

Reply:
Thanks, didn't notice that this layer was unnecessary. AddFull does require a NULL terminator on the va list too.